.github/workflows/docker-extension-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Build and publish docker extension
+on:
+  push:
+    tags:
+    - 'v*'
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: kinvolk/headlamp-extension
+jobs:
+  build_and_push_docker_extension:
+    name: Build docker extension
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Update docker image tags 
+      run: |
+        lastTag=$(git tag --list --sort=version:refname 'v*' | tail -2 | head -1)
+        latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+        echo "replacing ${lastTag} with ${latestTag}"
+        sed -i "s/$lastTag/$latestTag/g" ./docker-extension/Dockerfile
+        sed -i "s/$lastTag/$latestTag/g" ./docker-extension/docker-compose.yml
+    - name: Commit changes and push changes 
+      run: |
+          git config --global user.name 'docker-bot'
+          git config --global user.email 'sannagaraj@microsoft.com'
+          git diff
+          git add ./docker-extension/Dockerfile
+          git add ./docker-extension/docker-compose.yml
+          git status
+          lastTag=$(git tag --list --sort=version:refname 'v*' | tail -2 | head -1)
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          git commit --signoff -m "docker-ext:Update docker extension from $lastTag to $latestTag"
+          git log -1
+          git push origin main
+    - name: Install docker 
+      uses: docker-practice/actions-setup-docker@master
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Build and push docker extension image
+      run: |
+        docker buildx create --use
+        make docker-ext
```

Comment:
Should we push the changes to our repo in the `Commit changes...` step only after this step is successful?